### PR TITLE
Added sketch files for designers

### PR DESCRIPTION
**Pull request by sukhrajghuman**
on /download page

### CODE REVIEW

File: content/download/intro-designers.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: download/intro
 sidebar: intro-designers-sidebar.md
+paddingTop: true
+borderBottom: true
```
Comment (alex-page) [inline content/download/intro-designers.md]: We will need to do this a different way. We do not add css with props.

File: src/layout/download/intro-sidebar.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
+				text: link.title,
+				className: link.icon ? ` icon icon--${link.icon} ` : ''
+			}))
+		} />
 	</div>
 );
 
 IntroSidebar.propTypes = {
-
+	links: PropTypes.array,
```
Comment (alex-page) [inline src/layout/download/intro-sidebar.js]: Have a look at how we have done nested propTypes

File: src/layout/download/intro.js [hunk 3]
```diff
@@ -5,8 +5,10 @@ import React from 'react';
 /**
  * The Intro component
  */
-const DownloadIntro = ({ _body, borderTop, sidebar }) => (
-	<div className={`row download-intro${ borderTop ? ' download-intro--border' : '' }` }>
+const DownloadIntro = ({ _body, borderTop, sidebar, paddingTop, borderBottom }) => (
+	<div className={`row download-intro${ borderTop ? ' download-intro--border' : '' }
+	${ paddingTop ? ' download-intro--padding ' : ''}
+	${ borderBottom ? ' download-intro--border-bottom ' : ''}` }>
```
Comment (alex-page) [inline src/layout/download/intro.js]: This files changes should be reverted

File: src/sass/components/_download-intro.scss [hunk 4]
```diff
@@ -1,8 +1,16 @@
 .download-intro {
 	@include AU-space( margin-bottom, 2unit );
-	@include AU-space( padding-top, 4unit );
 }
 
 .download-intro--border {
 	@include AU-space( border-top, solid 0.25unit $AU-color-background-shade );
 }
+
```
Comment (alex-page) [inline src/sass/components/_download-intro.scss]: Revert these changes, we use lobotomised owls for spacing of consecutive elements.

File: src/sass/components/_icons.scss [hunk 5]
```diff
@@ -97,6 +97,14 @@
 	}
 }
 
+.icon--sketch {
+	&:before {
+		height: 1em;
+		width: 1em;
```
Comment (alex-page) [inline src/sass/components/_icons.scss]: Shouldn't need height and width, we also don't ever use `em`

File: content/download/intro-designers.md (hunk 1 above)
Comment (sukhrajghuman) [reply content/download/intro-designers.md]: fixed

File: src/layout/download/intro-sidebar.js (hunk 2 above)
Comment (sukhrajghuman) [reply src/layout/download/intro-sidebar.js]: fixed
